packages/gatsby/src/cache-dir/page-renderer.js

```diff
@@ -131,13 +131,13 @@ class PageRenderer extends React.Component {
       pathContext,
     }
 
-    const [replacementComponent] = apiRunner(`replaceComponentRenderer`, {
+    const replacementComponent = apiRunner(`replaceComponentRenderer`, {
```

Comment:
is this change valid? I think apiRunner returns array of responses and before the change it would pick first result from array

in master we use `pluginResponses[0]` to get that
https://github.com/gatsbyjs/gatsby/blob/a89057f003cf39dc1ca97a99120c368f32405872/packages/gatsby/cache-dir/component-renderer.js#L123-L127

Reply:
> in master we use pluginResponses[0] to get that

Thanks. That's right. I corrected it.